exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/configure_slc_repository.ipynb

```diff
@@ -0,0 +1,212 @@
+{
```

Comment:
<p>"If you <strong>choose</strong> ..." or "If you <strong>chose</strong> ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Fixed</p>